src/main/java/wolox/training/models/User.java

```diff
@@ -33,18 +47,29 @@
 
     @Column(nullable = false)
     private String username;
+    public void setUsername(String username){
```

Comment:
I think this setter could be private, because it's not used from another class (This way we could make this object even more immutable 😃 )